cmd/training-operator.v1/main.go

```diff
@@ -101,6 +106,11 @@ func main() {
 	flag.StringVar(&config.Config.MPIKubectlDeliveryImage, "mpi-kubectl-delivery-image",
 		config.MPIKubectlDeliveryImageDefault, "The image for mpi launcher init container")
 
+	// Cert generation flags
+	flag.IntVar(&webhookServerPort, "webhook-server-port", 9443, "Endpoint port for the webhook server.")
+	flag.StringVar(&webhookServiceName, "webhook-service-name", "training-operator-service", "Name of the Service used as part of the DNSName")
```

Comment:
training-operator-webhook-service ?

Reply:
This default service is not a dedicated service for the webhooks.
As a default, we use a common service.